src/model.rs

```diff
@@ -166,6 +184,78 @@ impl PodJob {
     }
 }
 
+fn serialize_pod_job<S>(pod_job: &PodJob, serializer: S) -> result::Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    serializer.serialize_str(&pod_job.hash)
+}
+
+fn deserialize_pod_job<'de, D>(deserializer: D) -> result::Result<PodJob, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    Ok(PodJob {
+        hash: String::deserialize(deserializer)?,
+        ..PodJob::default()
+    })
+}
+
+/// Result from a compute job run.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct PodResult {
+    /// Metadata that doesn't affect reproducibility.
+    #[serde(skip)]
+    pub annotation: Option<Annotation>,
+    /// Unique id based on reproducibility.
+    #[serde(skip)]
+    pub hash: String,
+    /// A pod job that originated the pod result.
+    #[serde(
+        serialize_with = "serialize_pod_job",
+        deserialize_with = "deserialize_pod_job"
+    )]
+    pub pod_job: PodJob,
+    /// Name given by orchestrator.
+    pub assigned_name: String,
+    /// State when terminated.
+    pub state: RunState,
+    /// Time in epoch when created in seconds.
+    pub created: u64,
```

Comment:
Perhaps consider using sered_with date time?

Reply:
Create and issue of doing custom seralization of datetime and throw it in utils.rs